storm/tests/test_storm.py

```diff
@@ -0,0 +1,644 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+from collections import defaultdict
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.storm import StormCheck
+from .common import (
+    TEST_STORM_CLUSTER_SUMMARY,
+    TEST_STORM_NIMBUSES_SUMMARY, TEST_STORM_SUPERVISOR_SUMMARY,
+    TEST_STORM_TOPOLOGY_SUMMARY, TEST_STORM_TOPOLOGY_RESP,
+    TEST_STORM_TOPOLOGY_METRICS_RESP
+)
+
+import pytest
+import responses
+import mock
+
+CHECK_NAME = 'storm'
+STORM_CHECK_CONFIG = {'instances': [{'server': 'http://localhost:8080', 'environment': 'test'}]}
+STORM_CHECK_INTEGRATION_CONFIG = {'instances': [{'server': 'http://localhost:8080', 'environment': 'integration'}]}
+
+
+def test_load_from_config():
+    check = StormCheck(STORM_CHECK_CONFIG, {})
+    check.update_from_config(STORM_CHECK_CONFIG['instances'][0])
+    assert('http://localhost:8080' == check.nimbus_server)
```

Comment:
We usually don't put parenthesis with assert. I tend to put the thing to assert on the left too?